lightning_transformers/core/nlp/config.py

```diff
@@ -2,8 +2,7 @@
 from pathlib import Path
 from typing import Optional, Union
 
-from lightning_transformers.core.config import TaskConfig
-from lightning_transformers.core.data import TransformerDataConfig
+from lightning_transformers.core.config import TaskConfig, TransformerDataConfig
```

Comment:
I would rather consider leveling it up
```suggestion
from lightning_transformers.core import TaskConfig, TransformerDataConfig
```
and adjust the `lightning_transformers/core/__init__.py` accordingly